scaffolder-templates/.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,72 @@
+name: Deploy the Pipeline trigger manifests workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      workflowId:
+        description: 'Workflow Id'
+        required: true
+        type: string
+      gitConfigUrl:
+        description: 'URL of git config repo'
+        required: true
+        type: string
+      tektonNamespace:
+        description: 'Namespace of Tekton resources'
+        required: true
+        type: string
+        default: 'sonataflow-infra'
+      argocdNamespace:
+        description: 'Namespace of ArgoCD resources'
+        required: true
+        type: string
+        default: 'argocd'
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Log in and set context
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT_SERVER }}
+          openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
+          namespace: default
+          insecure_skip_tls_verify: true
+
+      - name: Deploy Tekton trigger to the Kubernetes cluster
+        run: |
+          oc apply -n ${{ inputs.tektonNamespace }} -f tekton
+      
+      - name: Deploy ArgoCD application to the Kubernetes cluster
+        run: |
+          oc apply -n ${{ inputs.argocdNamespace }} -f argocd
+      - name: Create ArgoCD repository
+        run: |
+          echo "${{ secrets.SSH_PRIVATE_KEY }}" > sshPrivateKey
+          cat sshPrivateKey
```

Comment:
why this `cat` is needed?

Reply:
Just to create a security breach 😬 
Removing it